dev/index.html

```diff
@@ -26,5 +25,6 @@ <h6>Heading 6</h6>
     <section id="layouts">
 
     </section>
+    <script src="./index.js" type="module"></script>
```

Comment:
if the css didn't load when running the dev server, change this to `src="index.js"`